docs/examples/alpine/templates/alpine-pod.yaml

```diff
@@ -3,8 +3,8 @@ kind: Pod
 metadata:
   name: {{.Release.Name}}-{{.Chart.Name}}
   labels:
-    heritage: {{.Release.Service}}
-    chartName: {{.Chart.Name}}
+    heritage: {{.Release.Service | quote }}
```

Comment:
Will this catch the case where the user does `heritage: "{{.foo}}"`? That should be allowed, and is a workaround for things like `chart: "{{.Chart.Name}}-{{.Chart.Version}}"`